pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -1653,9 +1743,16 @@ def read_mwa_corr_fits(
 
             # rescale data
             # this needs to happen after the van vleck correction
-            self.data_array *= self.extra_keywords["SCALEFAC"]
+            if not mwax:
+                self.data_array *= self.extra_keywords["SCALEFAC"]
 
             # cable delay corrections
+            if correct_cable_len is None:
+                correct_cable_len = True
+                warnings.warn(
+                    "cable length correction is defaulted to True. To read in files \
+                    without applying the correction set correct_cable_len=False."
+                )
```

Comment:
I think this is good, but I think we want to identify that it's a change from before and a version where we'll remove this warning (v2.4?).